src/components/shared/ManagedTree.js

```diff
@@ -44,10 +44,6 @@ class ManagedTree extends Component {
       expanded: new Set(),
       focusedItem: null
     };
-
-    const self: any = this;
```

Comment:
`this.setExpanded` is used within `render` which is autoBound so this should work
`this.setExpanded` is used inside of `expandListItems()` which is called by `componentWillReceiveProps` (= autoBound)

So I think this is legit and should work. or am I completly wrong? @wldcordeiro @jasonLaster